statescript/statescript_test.go

```diff
@@ -199,17 +199,17 @@ func TestExecutor(t *testing.T) {
 	log.SetOutput(&buf)
 	fileP, err := createArtifactTestScript(tmpArt, "ArtifactInstall_Leave_00", "#!/bin/bash \necho 'error data' >&2")
 	assert.NoError(t, err)
-	res := execute(fileP.Name(), 100) // give the script plenty of time to run
-	assert.Equal(t, 0, res)
+	err = execute(fileP.Name(), 100) // give the script plenty of time to run
+	assert.Equal(t, nil, err)
 	assert.Contains(t, buf.String(), "error data")
 
 	buf.Reset()
 
 	// write more than 10KB to stderr
 	fileP, err = createArtifactTestScript(tmpArt, "ArtifactInstall_Leave_11", "#!/bin/bash \nhead -c 89999 </dev/urandom >&2\n exit 1")
 	assert.NoError(t, err)
-	res = execute(fileP.Name(), 100)
-	assert.Equal(t, 1, res)
+	err = execute(fileP.Name(), 100)
+	assert.NotEqual(t, nil, err)
```

Comment:
You can try `assert.EqualError(t, err, <message>)`, if I'm not mistaken, in this scenario the message will be `"exit status 1"`